src/index.js

```diff
@@ -0,0 +1,40 @@
+// import './styles.css';
```

Comment:
- Great Job for styling your page with Webpack Style/CSS Loader, I think you commented on the style import here which leads to not applying the style, so uncomment it 👍